binaryCalculation.java

```diff
@@ -0,0 +1,84 @@
+/*
+This Class is used to do all the 4 binaryCalulation for respective base
+Owner -> Rudra Pratap Singh
+Date -> 25/09/2024
+ */
+import java.text.DecimalFormat;
+public class binaryCalculation {
+    static DecimalFormat df = new DecimalFormat("#");
+    public static String addition(String firstInput, String secondInput, int base) {
+        try {
+            double firstNumber = binaryConversions.convertToDecimal(firstInput, base);
+            double secondNumber = binaryConversions.convertToDecimal(secondInput, base);
+            if (firstNumber == -1 || secondNumber == -1) {
+                return "";
+            }
+            double result = firstNumber + secondNumber;
+            if (Double.isInfinite(result)) {
+                return Constant.OVERFLOW_OCCURED;
+            }
+            String decimalResult = df.format(result);
+            return decimalResult;
+        } catch (ArithmeticException e) {
+            return Constant.OVERFLOW_OCCURED;
+        }
+    }
+
+    public static String subtraction(String firstInput, String secondInput, int base) {
+        try {
+            double firstNumber = binaryConversions.convertToDecimal(firstInput, base);
+            double secondNumber = binaryConversions.convertToDecimal(secondInput, base);
+            if (firstNumber == -1 || secondNumber == -1) {
+                return "";
+            }
+            double result = firstNumber - secondNumber;
+            if (Double.isInfinite(result)) {
+                return Constant.OVERFLOW_OCCURED;
+            }
+            String decimalResult = df.format(result);
+            return decimalResult;
+        } catch (ArithmeticException e) {
+            return Constant.OVERFLOW_OCCURED;
+        }
+    }
+
+    public static String multiplication(String firstInput, String secondInput, int base) {

```

Comment:
mension method's role , parameter and output in comments

Reply:
what is mension??